x-pack/test_serverless/shared/config.base.ts

```diff
@@ -159,7 +159,7 @@ export default async () => {
       try: 120 * 1000,
       waitFor: 20 * 1000,
       esRequestTimeout: 30 * 1000,
-      kibanaReportCompletion: 80 * 1000,
+      kibanaReportCompletion: 300 * 1000,
```

Comment:
All tests are running with a Mocha timeout of 6 minutes after which they fail in a bad way (no proper reporting etc since this is hitting the escape path and just kills the runner). The 5 minute timeout here brings the tests dangerously close to that limit. If we're sure that the other actions taken within one `it` are definitely always running in under one minute, we can go with this change. Otherwise, we should reduce the timeout here.

Reply:
> if we're sure that the other actions taken within one it are definitely always running in under one minute, we can go with this change


No, not sure. 

What I saw in the logs for those failures, is that it could take up to 5 minutes for the report to complete when the node decides to migrate. It might be even more, but it is usually less. 

can we increase the mocha timeout just for reporting tests to give it a bit more room?

Reply:
We can increase the mocha timeout only on config level. Since this test is in common and actually called from 3 different config files ([first](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/api_integration/test_suites/observability/common_configs/config.group1.ts), [second](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/api_integration/test_suites/search/common_configs/config.group1.ts), [thrid](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/api_integration/test_suites/security/common_configs/config.group1.ts)), we would need to increase the timeout in these three configs.

You could do that by adding something like this to the end of the config:
```javascript
    mochaOpts: {
      ...baseTestConfig.get('mochaOpts'),
      // increasing Mocha test timeout to 7 minutes for reporting tests
      timeout: 420000,
    },
```

Reply:
Discussed offline that can increase reporting timeout on a specific suite